Lib/_pyrepl/reader.py

```diff
@@ -52,7 +54,10 @@ def disp_str(buffer: str) -> tuple[str, list[int]]:
     b: list[int] = []
     s: list[str] = []
     for c in buffer:
-        if ord(c) < 128:
+        if c == '\x1a':
```

Comment:
Would it make sense to put `'\x1a'` in a constant like `CTRL_Z`so it's obvious what it represents?